### PR TITLE
THREAT-397 Reformat deep_get(event to event.deep_get(

### PR DESCRIPTION
**Pull request by akozlovets098**
### Changes

- Reformatted `deep_get(event` to `event.deep_get(`
- Reformatted `deep_walk(event` to `event.deep_walk(`

### CODE REVIEW

File: .scripts/mitre_mapping_check.py
Comment (akozlovets098) [inline .scripts/mitre_mapping_check.py:1]: This file was reformatted by black

**Comment by ben-githubs** [comment]
Some of the helper unit tests are failing because ImmutableDict doesn't have a deep_get field. I'm not sure of this is a problem with the test or the new changes. A couple of the failing tests have been due to us defining events as ImmutableDicts instead of PantherEvent, but I haven't worked through all of them. Please don't merge yet. :)

**Comment by ben-githubs** [comment]
The issue with the test is that the test setup functions were using `ImmutableCaseSentitiveDict` to simulate events. i changed those to PantherEvent objects to reflect the actual runtime environment.

I also replaced some `deep_get` calls with `get` where the call was only 1 layer deep (like Ariel mentioned in the meeting this morning).

**Comment by akozlovets098** [comment]
@ben-githubs Thanks for your help!
I fixed some linter issues and think that now the PR is ready for review

**Comment by arielkr256** [comment]
Very nice!  Does it make sense to refactor data_models as well?

**Comment by akozlovets098** [comment]
> Very nice! Does it make sense to refactor data_models as well?

I don't think we can. It appears that while using data models `PantherEvent` gets transformed into `ImmutableCaseInsensitiveDict` which doesn't have `deep_get` method

**Review by arielkr256 — APPROVED** [review]

**Pull request merged**